scripts/add-sibling.js

```diff
@@ -66,10 +68,3 @@ var indexPath = path.join(basePath, 'packages', 'all-packages', 'src', 'index.ts
 var index = fs.readFileSync(indexPath, 'utf8');
 index = index + 'import "' + package.name + '";\n';
 fs.writeFileSync(indexPath, index);
-
-// Add the extension to jupyterlab/package.json
-var jupyterlabPackagePath = path.join(basePath, 'jupyterlab', 'package.json');
```

Comment:
I confess to still being unclear on the intention of `jupyterlab/package.json` vs `jupyterlab/package.app.json`. If this is removed, does it need to be added elsewhere for the python package to pick up the new sibling?

In any event, a change here should be mirrored in `scripts/remove-sibling.js`.